src/Appwrite/GraphQL/Types/Mapper.php

```diff
@@ -109,9 +109,6 @@ public static function route(
                     'type' => $parameterType,
                     'description' => $parameter['description'],
                 ];
-                if ($parameter['optional']) {
-                    $params[$name]['defaultValue'] = $parameter['default'];
-                }
```

Comment:
why did we remove this ?

Reply:
this is handled by utopia framework now 👍🏻

Reply:
okay